src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CreateCaseCallbackService.java

```diff
@@ -100,16 +101,21 @@ public CreateCaseCallbackService(
                 caseId
             );
 
-            transformationClient.transformExceptionRecord(
+            SuccessfulTransformationResponse transformationResponse = transformationClient.transformExceptionRecord(
                 configItem.getTransformationUrl(),
                 exceptionRecord,
                 s2sTokenGenerator.generate()
             );
 
-            return Validation.valid(ImmutableMap.of("caseReference", UUID.randomUUID()));
+            return Validation.valid(new ProcessResult(
+                ImmutableMap.of("caseReference", UUID.randomUUID()),
+                transformationResponse.warnings
+            ));
         } catch (InvalidCaseDataException exception) {
-            // let controller deal with this. it is 422 or 400
-            throw exception;
+            return Validation.valid(new ProcessResult(
+                exception.getResponse().warnings,
+                exception.getResponse().errors
```

Comment:
Is it not Service will return a successful response even if there are warning/errors ? service need not throw an exception if say one of the field is missing/has incorrect value

Reply:
this PR does resolve exception throw and returns list of warnings/errors to the callback requester